portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,31 @@
 #content {
   margin-left: auto;
   margin-right: auto;
-  width: 650px;
+  width: 625px;
 }
 
-#greeting-container {
-  margin-top: 20px;
+#header {
+    padding: 30px;
+    background: #00CED1;
+    text-align: center;
 }
+
+#links {
+    padding: 20px;
+    display: inline-block;
+}
+
+footer {
+  width: 100%;
+  position:fixed;
+  bottom:0;
+  left:0;
+  text-align: center;
+  padding: 3px;
+  background-color: DarkSalmon;
+  color: white;
+}
+
+#center-alignment {
+    text-align: center;
+}
```

Comment:
nit: Could you add a newline at the end of the file?

Reply:
Yup